src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt

```diff
@@ -31,7 +31,8 @@ class LootOneNearbyPokestop(val sortedPokestops: List<Pokestop>, val lootTimeout
 
         if (nearbyPokestops.size > 0) {
             val closest = nearbyPokestops.first()
-            Log.normal("Looting nearby pokestop ${closest.id}")
+            val pokestopName = nearbyPokestops.first().details.name
```

Comment:
Maybe rename the settings.shouldDisplayWalkingToNearestUnused and use the same flag for both of these.